std/assembly/index.d.ts

```diff
@@ -304,6 +304,8 @@ declare namespace i8 {
   export const MIN_VALUE: i8;
   /** Largest representable value. */
   export const MAX_VALUE: i8;
+  /** Converts a string to an i8 of this type. */
+  export function parse(value: string, radix?: i32): i8;
```

Comment:
Perhaps a bit more concise:
```suggestion
  /** Parses a string as an i8. */
  export function parse(value: string, radix?: i32): i8;
```
Others below are similar.